[Mellanox] Update hw-management package to version 7.0010.2000

keboliu opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines:
https://github.com/Azure/SONiC/blob/gh-pages/CONTRIBUTING.md

** Make sure all your commits include a signature generated with `git commit -s` **

If this is a bug fix, make sure your description includes "fixes #xxxx", or
"closes #xxxx" or "resolves #xxxx"

Please provide the following information:
-->

**- Why I did it**

**Bug fixes**

-  In rare cases when thermal algorithem is Re-activiated after FAN/PSU insertion, FAN remians at high rpm
-  When stop hw-managment code received error in the log instead of exit code '0'.
-  In SPC1 i2c sometimes collide with chip reset coming from SDK
-  Remove raw eeprom data link, when working with PSU which don't have eeprom for "msn274x", "msn24xx" and "msn27xx" systems
-  Fix memory leak on mlxsw_core_bus_device module removal

**- How I did it**

Update the hw-mgmt version number in the make file
Update the hw-mgmt repo pointer

**- How to verify it**
run platform related test cases on all Mellanox platform

**- Which release branch to backport (provide reason below if selected)**

<!--
- Note we only backport fixes to a release branch, *not* features!
- Please also provide a reason for the backporting below.
- e.g.
- [x] 202006
-->

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012

**- Description for the changelog**
<!--
Write a short (one line) summary that describes the changes in this
pull request for inclusion in the changelog:
-->


**- A picture of a cute animal (not mandatory but encouraged)**

stephenxs commented:
```
admin@r-tigris-04:~$ sudo sfputil lpmode on Ethernet0
Enabling low-power mode for port Ethernet0...  OK
admin@r-tigris-04:~$ sudo sfputil lpmode off Ethernet0
Disabling low-power mode for port Ethernet0...  OK
```

keboliu committed:
[Mellanox] Update hw-management package to version 7.0010.2000

 - Bug fixes
    HW Mgmt core:
       In rare cases when thermal algorithem is Re-activiated after FAN/PSU insertion, FAN remians at high rpm
       When stop hw-managment code received error in the log instead of exit code '0'.
       In SPC1 i2c sometimes collide with chip reset coming from SDK

       Remove raw eeprom data link, when working with PSU which don't have eeprom for "msn274x", "msn24xx" and "msn27xx" systems
        *Kernel 4.19 - platform-x86-mlx-platform-Remove-PSU-EEPROM-configur.patch
        *Kernel 4.19 - platform-x86-mlx-platform-Remove-PSU-EEPROM-configur.patch

       Fix memory leak on mlxsw_core_bus_device module removal
        *Kernel 4.19 - mlxsw-core-Fix-memory-leak-on-module-removal.patch

Signed-off-by: Kebo Liu <kebol@nvidia.com>
